### PR TITLE
Meenu/Translation 75805 homepage live price

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +40,7 @@
         "gbimage-bridge": "^0.2.1",
         "i18next": "^21.6.5",
         "js-cookie": "^2.2.1",
+        "logs": "^1.5.10",
```
Comment (habib-deriv) [inline package.json:43]: BTW, do we need this @meenakshi-deriv ?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
         "@livechat/customer-sdk": "^3.1.0",
         "@loadable/component": "^5.15.2",
         "@svgr/webpack": "^6.2.1",
+        "@tanstack/react-table": "^8.7.4",
+        "@u-wave/react-vimeo": "^0.9.8",
```
Comment (habib-deriv) [inline package.json:13]: We don't need this too

File: src/pages/index.tsx [hunk 3]
```diff
@@ -85,7 +86,9 @@ const Home = () => {
                 has_organization_schema
             />
             <Hero />
-            <MarketsFold />
+
+            {is_eu ? <MarketsFold /> : <LivePricing />}
```
Comment (habib-deriv) [inline src/pages/index.tsx:90]: Hi @meenakshi-deriv, Remove this <MarketsFold /> as I mentioned earlier

File: package.json (hunk 2 above)
Comment (meenakshi-deriv) [reply package.json:13]: hi @habib-deriv we need this one `@tanstack/react-table": "^8.7.4",`

File: package.json (hunk 1 above)
Comment (meenakshi-deriv) [reply package.json:43]: I will remove this @habib-deriv

File: src/pages/home/live-pricing/components/_live_market_table.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    const requestMarketsData = useCallback(() => {
+        setIsLoading(true)
+
+        send(
+            {
+                trading_platform_asset_listing: 1,
+                platform: 'mt5',
+                type: 'brief',
+                region: region,
+                subscribe: 1,
```
Comment (habib-deriv) [inline src/pages/home/live-pricing/components/_live_market_table.tsx:50]:
```suggestion
                subscribe: 1,
```
```suggestion
```

File: package.json (hunk 2 above)
Comment (habib-deriv) [reply package.json]: Is this "@u-wave/react-vimeo": "^0.9.8", needed?

**Pull request closed without merging**